File: exercise2/statically.js
class Statically {
    //fix this code, so all of the instantiation and console.logs() work.  Try not to change the signature of the class, unless absolutely necessary.

    static #glue = "Epoxy";
    jello = "Jello";

    static moveAndShake() {
        console.log("Never stagnate in life.  Be agile, change, and improve...");
    }

    getGlue(){
        return Statically.#glue;
    }


    static stubborn() {
        this.moveAndShake();
        console.log("Please don't make me change... I just don't like it...");
    }

    static explain() {
        Statically.stubborn();
        console.log(this.#glue);
        console.log("Static in both languages makes the method or variable globally accessible but in Java it doesn't need to be instanced to be called while in JS you have to call the class itself to run the method.")
        //write an explanation for how the keyword static behaves differently in javascript vs. java.
    }

}

//get the below code to run correctly without errors:  Make sure you explain what you did in the the explain() method, defined in the class.  Do NOT remove the 'static' keyword from the explain() method.

const stats = new Statically();

Statically.moveAndShake();
Statically.stubborn();
Statically.explain();

console.log(stats.getGlue());
console.log(stats.jello);

//Once you have finished getting statistically.js to run, refactor the code here in Java.

File: exercise2/Statically.java
package exercise2;

public class Statically {
    //Once you have finished getting statistically.js to run, refactor the code here in Java.
    //fix this code, so all of the instantiation and console.logs() work.  Try not to change the signature of the class, unless absolutely necessary.

    private static String glue = "Epoxy";
    public static String jello = "Jello";

    public static void moveAndShake() {
        System.out.println("Never stagnate in life.  Be agile, change, and improve...");
    }

    public static String getGlue(){
        return glue;
    }


    public static void stubborn() {
        moveAndShake();
        System.out.println("Please don't make me change... I just don't like it...");
    }

    public static void explain() {
        stubborn();
        System.out.println(glue);
        System.out.println("Static in both languages makes the method or variable globally accessible but in Java it doesn't need to be instanced to be called while in JS you have to call the class itself to run the method.");
        //write an explanation for how the keyword static behaves differently in javascript vs. java.
    }

    public static void main(String[] args){
        moveAndShake();
        stubborn();
        explain();

        System.out.println(getGlue());
        System.out.println(jello);
    }

}
